docs/hosting/server-setups/digital-ocean.md

```diff
@@ -0,0 +1,178 @@
+# Hosting n8n on DigitalOcean
+
+This hosting guide shows you how to self-host n8n on a DigitalOcean droplet. It uses n8n and [Caddy](http://caddyserver.com){:target="_blank" class=.external-link} (a reverse proxy) to allow access to the Droplet from the internet. It uses [Docker Compose](https://docs.docker.com/compose/){:target="_blank"} to create and define the application components and how they work together.
+
+## Create a Droplet
+
+Log in to DigitalOcean and select the project to host the Droplet and click **Droplets** from the **Manage** menu. You can use [the Docker image](https://marketplace.digitalocean.com/apps/docker){:target="_blank" class=.external-link} from the **Marketplace** tab or use one of the Linux distributions and [install Docker yourself](https://www.docker.com/get-started/){:target="_blank" class=.external-link}. The rest of this tutorial assumes you have Docker installed on the Droplet.
+
+## Log in to your Droplet
+
+The remainder of the steps in this guide require you to log in to the Droplet using a terminal with SSH. [Find more details on how to do this in the Digital Ocean documentation](https://docs.digitalocean.com/products/droplets/how-to/connect-with-ssh/){:target="_blank"}.
+
+## Create folders and files
+
+Both n8n and Caddy require creating folders that the host operating system (the DigitalOcean Droplet) copies to Docker containers to make them available to Docker.
+
+Create the following on the Droplet in your home folder:
+
+- `caddy_config`: Holds the Caddy configuration files.
+- `caddy_data`: A cache folder for Caddy.
+- `local_files`: A folder for files you upload or add via n8n.
+
+Use the following command to create the folders:
+
+```shell
+mkdir caddy_config caddy_data local_files
```

Comment:
Should we also tell them to create an n8n directory? The subsequent .env file expects everything to be in root/n8n, not just home dir